src/components/Divider.tsx

```diff
@@ -0,0 +1,19 @@
+// copied from Azure Data Explorer plugin since there is not Divider component in G<10.1
+import React from 'react';
+// import { Divider as GrafanaDivider} from '@grafana/ui';
```

Comment:
wouldn't this throw an error if we are running the plugin in grafana < 10.1? my understanding is that the plugins rely on the runtime for the `@grafana/ui` import so if we run in grafana < 10.1 it would not find the `Divider` component.

also, i think it'd be good to remove the commented out code for now and leave it in a draft branch and then link it to the github issue for enabling this.

Reply:
FWIW I tested this when I implemented and it didn't throw in G<10.1 but maybe worth someone else checking too 😊